### PR TITLE
Issue/6

### PR DESCRIPTION
**Pull request by Joaquim-Ferrer**

### CODE REVIEW

File: dotworks/dotworks_server/forms.py [hunk 1]
```diff
@@ -108,3 +108,20 @@ class InternshipCreationForm(forms.Form):
 
 class InscriptionAddForm(forms.Form):
     pass
+
+class ChangePasswordForm(forms.Form):
+    password = forms.CharField(
+        widget=forms.PasswordInput(),
+        label="Password antiga",
```
Comment (mmachado95) [inline dotworks/dotworks_server/forms.py]: We need to start using english. So it's best to already change labels to english

File: dotworks/dotworks_server/forms.py [hunk 2]
```diff
@@ -108,3 +108,20 @@ class InternshipCreationForm(forms.Form):
...
+        required=True
+    )
+    new_password = forms.CharField(
+        widget=forms.PasswordInput(),
+        label="Nova password",
```
Comment (mmachado95) [inline dotworks/dotworks_server/forms.py]: Same as above

File: dotworks/dotworks_server/forms.py [hunk 3]
```diff
@@ -108,3 +108,20 @@ class InternshipCreationForm(forms.Form):
...
+        required=True
+    )
+    confirm_new_password = forms.CharField(
+        widget=forms.PasswordInput(),
+        label="Confirmar nova password",
```
Comment (mmachado95) [inline dotworks/dotworks_server/forms.py]: Same as above

File: dotworks/dotworks_server/templates/change_password_page.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <title>.works</title>
+</head>
+
+<body>
+    <form action="{% url 'change_password' %}" method="post">
+        {% csrf_token %}
+        <h2>Change you password:</h2>
+        {{changePasswordForm}}
+        <br><br>
+        <input type="submit" value="Submeter" />
```
Comment (mmachado95) [inline dotworks/dotworks_server/templates/change_password_page.html:11]: Same as above

**Review by mmachado95 — CHANGES_REQUESTED** [review]
Only need to change the Portuguese to English.

**Comment by Joaquim-Ferrer** [comment]
Done.

**Pull request merged**